spec/rest-catalog-open-api.yaml

```diff
@@ -1002,7 +1009,9 @@ paths:
                 TableToLoadDoesNotExist:
                   $ref: '#/components/examples/NoSuchTableError'
         409:
-          description: Conflict - Polaris has processed a newer notification timestamp for the table.
+          description:
+            Conflict - The timestamp of the notification being sent is older than or equal to the 
```

Comment:
Is it really `the notification being sent` or is it `the received notification`?

Reply:
Should be the latter, updated the description 🙂